admin_manual/release_notes.rst

```diff
@@ -74,7 +74,8 @@ The Password Policy Extension
 .. note::
    These features don't apply to users imported from LDAP or other backends but only for local users created by administrators or the `Guests <https://marketplace.owncloud.com/apps/guests>`_ extension.
 
-Imposing password expiration and history policies enhance security. For example, when users are forced to choose a new password, they can be blocked from using their previous passwords (the password history count is configurable). However, increasing security can also decrease user satisfaction, such as forcing them to change their password upon first login, or change their passwords too frequently (such as every 90 days). That said, we encourage an acceptable balance between security and usability.
+Imposing password expiration and history policies enhance security. For example, when users are forced to choose a new password, they can be blocked from using their previous passwords (the password history count is configurable). To ensure a good experience it is possible to inform users via email and internal notifications (web interface and clients) a configurable amount of days before the password expires and when the password has expired.
+However, increasing security can also decrease user satisfaction, such as forcing them to change their password upon first login, or change their passwords too frequently (such as every 90 days). That said, we encourage an acceptable balance between security and usability.
```

Comment:
- "too frequently (...)" => "too frequently like for example every 90 days"

You could also reverse the sentence and start with the essential part: "We encourage an acceptable balance between security and usability by choosing sensible values for password expiration. A high frequency of password changes might increase security but could also decrease user satisfaction." (or cause user frustration)